client/example_object_test.go

```diff
@@ -0,0 +1,271 @@
+
+	// Configure get parameters
+	var prm client.PrmObjectGet
+
+	// Initiate object retrieval
+	objHeader, reader, err := c.ObjectGetInit(ctx, containerID, objectID, signer, prm)
+	if err != nil {
+		switch {
+		case errors.Is(err, apistatus.ErrObjectAccessDenied):
+			var errAccessDenied apistatus.ObjectAccessDenied
+			if errors.As(err, &errAccessDenied) {
+				reason := errAccessDenied.Reason()
+				if reason != "" {
+					log.Fatal(fmt.Errorf("access denied: %s", reason))
+				} else {
+					log.Fatal(fmt.Errorf("access denied: %w", err))
+				}
+			}
+		case errors.Is(err, apistatus.ErrObjectNotFound):
+			log.Fatal(fmt.Errorf("object not found: %w", err))
+		default:
+			log.Fatal(fmt.Errorf("init object get: %w", err))
+		}
+	}
+	defer reader.Close()
+
+	// Access object header information
+	fmt.Printf("Object owner: %s\n", objHeader.Owner())
+	fmt.Printf("Payload size: %d bytes\n", objHeader.PayloadSize())
+	fmt.Println("Attributes:")
+	for _, v := range objHeader.Attributes() {
+		fmt.Printf("\t%s: %s\n", v.Key(), v.Value())
+	}
+	// Example output:
+	// Object owner: NfeCA7AuP5zodtJEPsSdkZ2gB3hKdNEhUg
+	// Payload size: 1024 bytes
+	// Attributes:
+	//	FileName: example.txt
+	//	ContentType: text/plain
+
+	// Read the payload
+	payload, err := io.ReadAll(reader)
+	if err != nil {
+		log.Fatal(fmt.Errorf("read payload: %w", err))
+	}
+
+	fmt.Printf("Successfully read %d bytes\n", len(payload))
+	// Example output: Successfully read 1024 bytes
+}
+
+// ExampleClient_ObjectPutInit demonstrates how to initialize an object upload
+// operation using the NeoFS SDK for Go.
+func ExampleClient_ObjectPutInit() {
+	ctx := context.Background()
+
+	// Load wallet and decrypt account
+	// This is required to sign the upload request.
+	w, err := wallet.NewWalletFromFile(walletPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	addr := w.GetChangeAddress()
+	acc := w.GetAccount(addr)
+	err = acc.Decrypt(walletPass, keys.NEP2ScryptParams())
+	if err != nil {
+		log.Fatal(err)
+	}
+	signer := user.NewAutoIDSigner(acc.PrivateKey().PrivateKey)
+
+	// Create and configure client
+	var prmInit client.PrmInit
+	c, err := client.New(prmInit)
+	if err != nil {
+		log.Fatal(fmt.Errorf("client init: %w", err))
+	}
+
+	// Connect to NeoFS node
+	var prmDial client.PrmDial
+	prmDial.SetServerURI(testnetStorageNode)
+	// Optional settings
+	prmDial.SetTimeout(15 * time.Second)
+	prmDial.SetStreamTimeout(15 * time.Second)
+
+	if err = c.Dial(prmDial); err != nil {
+		log.Fatal(fmt.Errorf("dial: %w", err))
+	}
+
+	// Parse container ID from string
+	containerID, err := cid.DecodeString(cnrIDStr)
+	if err != nil {
+		log.Fatal(fmt.Errorf("parse container ID: %w", err))
+	}
+
+	// Create object header with required fields
+	var obj object.Object
+	obj.InitCreation(object.RequiredFields{
+		Container: containerID,
+		Owner:     signer.UserID(),
+	})
+
+	// Optional: set custom attributes
+	obj.SetAttributes(
+		object.NewAttribute(object.AttributeFileName, "example.txt"),
```

Comment:
How about `FilePath` with `path/to/example.txt`? Both are used often and it's important to show the difference in intended use.